src/compas/geometry/_core/intersections.py

```diff
@@ -382,6 +382,37 @@ def intersection_segment_plane(segment, plane, tol=1e-6):
 
     return None
 
+def intersection_polyline_plane(polyline, plane, expected_number_of_intersections=2, tol=1e-6):
+    """ Calculates the intersection point of a plane with a polyline. 
+        returns a list of intersections 
+        by default it will allow two intersections. reduce expected_number_of_intersections to speed up.
```

Comment:
Please remove whitespace between the triple quotes and the summary. Please also remove the indents of the second and third line.

Reply:
I will apply this to everything, thank you very much